scripts/run-metadata-validation.ps1

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+<#
+.Synopsis
+    Runs MetadataParser tool to validate latest XSLT rules don't break downstream parsing of the metadata
+
+.Description
+    1. tranforms latests snapshots of beta and v1 metadata
+    2. validates that the transformed metadata is parsable as OData EDM model and conversion to OpenAPI document is possible
+
+.Example
+    ./scripts/run-metadata-validation.ps1 -repoDirectory C:/github/msgraph-metadata
+
+.Example
+    ./scripts/run-metadata-validation.ps1 -repoDirectory $GITHUB_WORKSPACE
+
+.Parameter repoDirectory
+    Full path the the root directory of msgraph-metadata checkout.
+#>
+
+param(
+    [Parameter(Mandatory=$true)][string]$repoDirectory
+)
+
+$transformCsdlDirectory = Join-Path $repoDirectory "transforms/csdl"
```

Comment:
```suggestion
$transformCsdlDirectory = Join-Path $repoDirectory "transforms" "csdl"
```

Reply:
Unfortunately `Join-Path` doesn't work with varargs and you need [weird tricks in piping](http://codebuckets.com/2016/08/20/powershell-join-path-with-multiple-parameters/) or multiple levels of function composition to get it working in single line.

Forward slash works cross platform.